src/chains/RouteMatchingProcessorChain.ts

```diff
@@ -15,7 +15,7 @@ export class RouteMatchingProcessorChain extends ProcessorChain implements IRequ
    private readonly _matcher: RegExp;
    private readonly _paramKeys: PathToRegexpKey[] = [];
 
-   public constructor(subprocessors: ErrorHandlingRequestProcessor[], path: PathParams, caseSensitive: boolean = false, method?: string) {
+   public constructor(subprocessors: ErrorHandlingRequestProcessor[], path: PathParams, method?: string, caseSensitive: boolean = false) {
```

Comment:
I think this change should be OK since we don't export RouteMatchingProcessorChain in the public API and it doesn't exist in the actual Express API. But I'd like to get a second opinion from @jthomerson